_data/translations.yml

```diff
@@ -40,4 +40,12 @@ cookieConsent:
 cookieConsentApprove:
   en: "Approve"
   pl: "Zgadzam się"
-
+joinUs:
+  en: "Join us"
```

Comment:
Czy jest sens wyświetlania tego w wersji angielskiej skoro ogłoszenia mamy tylko po polsku i raczej kierujemy je do ludzi, którzy mówią po polsku?

Reply:
@adampij wątpliwość biznesowa

Reply:
Całkiem spora część odbiorców bloga stanowią osoby z polski, które mają ustawiony angielski język w przeglądarce. Zakładam że jeżeli jakiś anglik kliknie na link, zobaczy że część z ofertami jest po ang i się zniechęci, to nic się nie stanie.